### ISSUE TITLE
Interested in project

### ISSUE THREAD

**Issue by icepicknz** [schwark/smartthings-tesla#1]
Hi, i'm interested in your project plan and details; wish to share? :)

I see there is a full API available at https://timdorr.docs.apiary.io/#
My end game is to get smart things WebCoRe to turn charging on or off depending on my solar production which Im trying to find a tesla device type that can do this on/off charging option

**Comment by schwark** [comment]
yeah - I am planning to implement that API in groovy to get this to work.. :)
